### DISCUSSION TITLE
Obsessive-compulsive disorder | 强迫症

### DISCUSSION THREAD

**Discussion by insvl1129** [t1m0thyj/WinDynamicDesktop#362] [General]
![image](https://user-images.githubusercontent.com/67249825/127728885-16489d64-20e4-4164-9cef-87ab9c580bcc.png)

**Comment by t1m0thyj** [comment]
Which translation are you using?

**Reply by RenzheLee** [reply]
Chinese,this is Chinese

**Comment by RenzheLee** [comment]
咱也不能太严苛要求老外对吧，中文毕竟是最难学的，能用就可以啦
